eth/core/processor.go

```diff
@@ -124,6 +118,7 @@ func (sp *StateProcessor) Prepare(evm *vm.GethEVM, header *types.Header) {
 	// *technically* the precompiles change based on the chain config rules, to be fully correct,
 	// we should check every block.
 	sp.BuildAndRegisterPrecompiles(precompile.GetDefaultPrecompiles(&rules))
+	sp.BuildAndRegisterPrecompiles(sp.pp.GetPrecompiles(&rules))
```

Comment:
only need to re-register the geth stateless precompiles every block

Reply:
will be needed (at least for now), in my geth tx pool branch